package.json

```diff
@@ -1147,7 +1147,6 @@
     "chroma-js": "^2.1.0",
     "chrome-remote-interface": "^0.33.3",
     "classnames": "2.2.6",
-    "color": "^4.2.3",
```

Comment:
minor nit: since these are removed from here, we can also clear these up from the `renovate.json`

Reply:
Created a follow up here: https://github.com/elastic/kibana/pull/221603